packages/node-resolve/src/index.js

```diff
@@ -121,30 +121,17 @@ export function nodeResolve(opts = {}) {
       return false;
     }
 
-    const importSpecifierList = [];
+    const importSpecifierList = [importee];
 
     if (importer === undefined && !importee[0].match(/^\.?\.?\//)) {
       // For module graph roots (i.e. when importer is undefined), we
       // need to handle 'path fragments` like `foo/bar` that are commonly
       // found in rollup config files. If importee doesn't look like a
       // relative or absolute path, we make it relative and attempt to
-      // resolve it. If we don't find anything, we try resolving it as we
-      // got it.
+      // resolve it.
       importSpecifierList.push(`./${importee}`);
     }
 
-    const importeeIsBuiltin = builtins.has(importee);
-
-    if (importeeIsBuiltin) {
-      // The `resolve` library will not resolve packages with the same
-      // name as a node built-in module. If we're resolving something
-      // that's a builtin, and we don't prefer to find built-ins, we
-      // first try to look up a local module with that name. If we don't
-      // find anything, we resolve the builtin which just returns back
-      // the built-in's name.
-      importSpecifierList.push(`${importee}/`);
```

Comment:
all tests pass without this now, so I think maybe this is no longer needed since the other work I did to add builtin support to `resolve`